### ISSUE TITLE
beer factories wont open 

### ISSUE THREAD

**Issue by luciferakathedevil** [wol-soft/beerplop#121]
after I got the proxy upgrade for the storage I sacrificed and now I can't even open the beer factories, nor close it after it pops up it's just a blank screen so I have to refresh to play the game

**Comment by wol-soft** [comment]
Hi @luciferakathedevil, did you equip the proxy upgrade or did the issue occur directly after finishing the upgrade?
It's a display error which I try to reproduce to fix the issue.

Relevant error log:

```
Client Error [beerplop@1.69.1]
TypeError: Cannot read property 'hasProjectQueue' of undefined
    at Render.<anonymous> (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:429477)
    at Array.map (<anonymous>)
    at Render.<anonymous> (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:428950)
    at Array.map (<anonymous>)
    at Render.renderFactoriesMap (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:427338)
    at Render.renderOverlay (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:423616)
    at OverlayController.openOverlay (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:250260)
    at HTMLDivElement.<anonymous> (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:32:249494)
    at HTMLDivElement.dispatch (https://wol-soft.de/apps/beerplop/dist/js/beerplop-game-1.69.1.min.js:2:4...',
referer: https://wol-soft.de/apps/beerplop/plop
```

**Comment by wol-soft** [comment]
Alright, I've reproduced and fixed the issue and patched it with version 1.69.2
